ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py

```diff
@@ -390,8 +390,17 @@
         version_added: "1.8.0"
       event:
         description:
-          - Events on which conditions such as SSL Extension/tcp can be triggered.
+          - Events on which conditions such as c(ssl_extension) / c(tcp)  can be triggered.
+          - All options are supported with c(tcp),c(ssl_extension)
         type: str
+        choices:
+            - request
+            - client_accepted
+            - proxy_connect
+            - proxy_request
+            - proxy_response
+            - ssl_client_hello
+            - ssl_client_server_hello_send
```

Comment:
are you making event parameter to have choices? you cannot specify this this way you will get sanity errors, also event() parameter must not have choices added to it as it will break for users. See what I did in such situation and update accordingly: 

https://github.com/F5Networks/f5-ansible/blob/ff845ea507c892abc549309ad5fd1f456187e018/ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py#L93